optaplanner-examples/src/main/java/org/optaplanner/examples/meetingscheduling/persistence/MeetingSchedulingXlsxFileIO.java

```diff
@@ -317,6 +329,45 @@ private void readMeetingDuration(Meeting meeting) {
                     .collect(toList());
         }
 
+        private TimeGrain extractTimeGrain(Meeting meeting, Map<LocalDateTime, TimeGrain> timeGrainMap) {
+            String dateString = nextStringCell().getStringCellValue();
+            String startTimeString = nextStringCell().getStringCellValue();
+            if (!dateString.isEmpty() || !startTimeString.isEmpty()) {
```

Comment:
As both dateString and startTimeString are used in this branch, is the logical OR correct here?

Reply:
Yes, so it fails fast if only one them is filled in.